xml/System.Threading.Tasks/ValueTask`1.xml

```diff
@@ -392,7 +394,15 @@ Note: The use of the <see cref="ValueTask{TResult}"/> type is not supported by V
       <Docs>
         <summary>Gets the result.</summary>
         <value>The result.</value>
-        <remarks></remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+## Remarks  
+ Accessing the property's get accessor blocks the calling thread until the asynchronous operation is complete; it is equivalent to calling the <xref:System.Threading.Tasks.Task.Wait%2A> method.  
+  
+ Once the result of an operation is available, it is stored and is returned immediately on subsequent calls to the <xref:System.Threading.Tasks.ValueTask%601.Result%2A> property. Note that, if an exception occurred during the operation of the task, or if the task has been cancelled, the <xref:System.Threading.Tasks.ValueTask%601.Result%2A> property does not return a value. If you attempt to access the property value in this situation, the <xref:System.Threading.Tasks.ValueTask%601.Result%2A> property does not throw an <xref:System.AggregateException> exception as the <xref:System.Threading.Tasks.Task%601.Result%2A> property does.  
```

Comment:
I think it doesn't make sense to directly copy the text from `Task<T>.Result`, because `ValueTask` does not have `Wait` and because `ValueTask` does not itself do all that result caching.

I would write it like this:

> If this \<xref:System.Threading.Tasks.ValueTask%601> wraps a successful result, this property directly returns it.
>
>  If it wraps a \<xref:System.Threading.Tasks.Task%601>, the behavior of \<xref:System.Threading.Tasks.ValueTask%601.Result%2A> is similar to the behavior of accessing \<xref:System.Threading.Tasks.Task%601.Result%2A> on the wrapped task: if the task hasn't completed, accessing the property blocks until it completes; if the task completed successfully, the property returns the result; if the task completed unsuccessfully, accessing the property throws. The thrown exception is not wrapped in an \<xref:System.AggregateException>, which is different from the behavior of \<xref:System.Threading.Tasks.Task%601.Result%2A> in the same situation.